lib/BuildSystem/BuildSystem.cpp

```diff
@@ -846,15 +846,18 @@ class DirectoryContentsTask : public Task {
     }
 
     std::vector<std::string> filenames;
-    getContents(path, filenames);
+    if (getContents(path, filenames)) {
+      ti.complete(BuildValue::makeFailedInput().toData());
```

Comment:
@dmbryson Is `makeFailedInput` a reasonable `BuildValue` to use for the failure case here?

Reply:
Seems reasonable to me.